### PR TITLE
Support to GoCommerce logo

### PR DESCRIPTION
**Pull request by gustavorosolem**
#### What is the purpose of this pull request?
Show the correct logo for GoCommerce store.

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/FooterVtexLogo.js [hunk 1]
```diff
@@ -18,7 +20,7 @@ const FooterVtexLogo = ({ logoUrl, imageSrc }) => {
         <img className={`${footer.logoImage} h3`} src={logoUrl} />
       </span>
       <span className={`${footer.badge} pa2-s pa1-ns nt7-ns`}>
-        <img className={`${footer.vtexLogoItem} h3 w3`} src={imageSrc} />
+        <img className={`${footer.vtexLogoItem} ${isPlatformGC(account) ? 'w4' : 'h3 w3'}`} src={imageSrc} />
```
Comment (theomoura) [inline react/components/FooterVtexLogo.js]: You could use classNames here

File: react/index.js [hunk 2]
```diff
@@ -280,7 +281,11 @@ export default class Footer extends Component {
                 </p>
               ))}
           </div>
-          <FooterVtexLogo logoUrl={logo} />
+          <RenderContextConsumer>
```
Comment (luciannojunior) [inline react/index.js]: Use `withRuntimeContext` instead. It provides the _account_ :)

File: react/images/GoCommerce.svg [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="158px" height="32px" viewBox="0 0 158 32" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: sketchtool 52.2 (67145) - http://www.bohemiancoding.com/sketch -->
+    <title>2BF30658-B4D8-4F35-BCBC-5CD1D2A491C5</title>
+    <desc>Created with sketchtool.</desc>
```
Comment (luciannojunior) [inline react/images/GoCommerce.svg]: Use https://jakearchibald.github.io/svgomg/

File: react/images/GoCommerce-BW.svg [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="158px" height="32px" viewBox="0 0 158 32" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: sketchtool 52.2 (67145) - http://www.bohemiancoding.com/sketch -->
```
Comment (luciannojunior) [inline react/images/GoCommerce-BW.svg]: Use https://jakearchibald.github.io/svgomg/

File: CHANGELOG.md [hunk 5]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+ - Support to GoCommerce logo
```
Comment (luciannojunior) [inline CHANGELOG.md]: Add a final dot

File: react/components/FooterVtexLogo.js [hunk 6]
```diff
@@ -4,10 +4,12 @@ import PropTypes from 'prop-types'
 import withImage from './withImage'
 import footer from '../footer.css'
 
+const isPlatformGC = account => account.indexOf('gc_') === 0 || account.indexOf('gc-') === 0
```
Comment (luciannojunior) [inline react/components/FooterVtexLogo.js]: Use [Ramda's includes](https://ramdajs.com/docs/#includes) instead

File: react/components/FooterVtexLogo.js [hunk 7]
```diff
@@ -27,13 +29,14 @@ const FooterVtexLogo = ({ logoUrl, imageSrc }) => {
 FooterVtexLogo.displayName = 'FooterVtexLogo'
 
 FooterVtexLogo.propTypes = {
+  account: PropTypes.string,
```
Comment (luciannojunior) [inline react/components/FooterVtexLogo.js]: Add a doc to maintain conciseness.

File: react/index.js (hunk 2 above)
Comment (luciannojunior) [reply react/index.js]: In the `FooterVtexLogo`

Comment (gustavorosolem) [reply react/index.js]: I still prefer the use of Render Props, they provide a better understanding of what the component is receiving.
But if the use of the HOC is a definition, it's ok either.

File: react/components/FooterVtexLogo.js (hunk 6 above)
Comment (gustavorosolem) [reply react/components/FooterVtexLogo.js]: This don't make sense to me due to Ramda perform a indexOf too, but ok...
https://github.com/ramda/ramda/blob/v0.26.1/source/internal/_includes.js
https://github.com/ramda/ramda/blob/v0.26.1/source/internal/_indexOf.js

**Comment by gustavorosolem** [comment]
@luciannojunior Can you please review again?

**Pull request merged**